KlingonAssistant/data/mem-05-H.xml

```diff
@@ -1239,15 +1239,15 @@ A physical path would be a {taw:n}.</column>
       <!-- Checked on: 2012.12.01. -->
       <column name="_id">16140</column>
       <column name="entry_name">HeghmoH</column>
-      <column name="part_of_speech">v:is,deriv</column>
+      <column name="part_of_speech">v:t_c,deriv</column>
       <column name="definition">be fatal</column>
-      <column name="definition_de">tödlich</column>
+      <column name="definition_de">tödlich sein, töten</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{HoH:v}</column>
-      <!-- Is this actually the same as {Hegh:v} with the suffix {-moH:v}? -->
-      <column name="notes"></column>
-      <column name="notes_de"></column>
+      <!-- Is this actually the same as {Hegh:v} with the suffix {-moH:v}? Edit by Lieven: I'm sure it is, as the latest discussion on the ML shows.-->
+      <column name="notes">See {qawmoH:v} for an example of how to use a verb like this one.</column>
```

Comment:
I think this comment is more likely to confuse than clarify here. Unlike {qawmoH}, or {SovmoH} or {tuQmoH} (which have this note), the root verb here doesn't "usually" take an object. I say "usually" because apparently something like {SuvwI' Hegh Hegh} "he died a warrior's death" is possible (based on {tlhIngan yIn DayIn}), so I suppose one could say something like {QughvaD SuvwI' Hegh HeghmoH qIrq} "Kirk caused Kruge to die a warrior's death". But typically one would just say {Qugh HeghmoH qIrq}.